integration/type-registry/bar/bar.ts

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+import { messageTypeRegistry } from '../typeRegistry';
+import { util, configure, Writer, Reader } from 'protobufjs/minimal';
+import * as Long from 'long';
+import { Foo } from '../foo';
+
+export const protobufPackage = 'foo.bar';
+
+export interface Bar {
+  $type: 'foo.bar.Bar';
+  foo: Foo | undefined;
+}
+
+const baseBar: object = { $type: 'foo.bar.Bar' };
+
+export const Bar = {
```

Comment:
I think this object should have a `$type` field as well

Reply:
Sgtm

Reply:
Added the `$type` field to message statics.

This should later help with `google.protobuf.Any` helpers:

```ts
const anyMessage: Any = ...;

if (Any.is(anyMessage, Foo)) {
  const foo: Foo = Any.unpack(anyMessage, Foo);
}
```

https://github.com/protocolbuffers/protobuf/blob/v3.15.6/src/google/protobuf/any.proto#L67-L75

Reply:
Nice!